src/RA_Dlg_MemBookmark.cpp

```diff
@@ -23,10 +24,9 @@ const int COLUMN_WIDTH[] = { 112, 64, 64, 64, 54 };
 static_assert(SIZEOF_ARRAY(COLUMN_TITLE) == SIZEOF_ARRAY(COLUMN_WIDTH), "Must match!");
 }
 
-const COMDLG_FILTERSPEC c_rgFileTypes[] =
-{
-    { L"Text Document (*.txt)",       L"*.txt" }
-};
+// If it's only going to be one we could make it a unique_ptr
```

Comment:
Comments like this should be in the PR, not the code.

The method expects a pointer to a list. Just because the list only has one item doesn't mean that we should use an alternate data type and pretend it's a list.

Additionally, leaving it as a list makes it easier to add additional items in the future if desired.

Reply:
Thank you. From what you're saying, it should be std::array since it's a global. We could always add more later if needed.